dsf-fhir/dsf-fhir-server/src/test/resources/integration/task-bundle.json

```diff
@@ -91,7 +91,7 @@
 				"address": [
 					{
 						"line": [
-							"Highmedstra�e 12"
+							"Highmedstra�e 12"
```

Comment:
check encoding

Reply:
Practitioner and PractitionerRole can be removed from the test bundle since they are optional in the feasibility request.